content/docs/setup/kubernetes/helm-install.md

```diff
@@ -97,6 +97,7 @@ following table:
 | `global.tag` | Specifies the TAG for most images used by Istio | valid image tag | `0.8.0` |
 | `global.proxy.image` | Specifies the proxy image name | valid proxy name | `proxyv2` |
 | `global.proxy.includeIPRanges` | Specifies the IP ranges for which outbound traffic is redirected to Envoy | List of IP ranges in CIDR notation separated by the escaped comma `\,` . Use `*` to redirect all outbound traffic to Envoy | `*` |
+| `global.proxy.envoyStatsd` | Specifies the Statsd server that Envoy should send its stats to | host/IP and port | `istio-statsd-prom-bridge:9125` |
 | `global.imagePullPolicy` | Specifies the image pull policy | valid image pull policy | `IfNotPresent` |
```

Comment:
@ymesika note I am in the process of automating the production of this table here:
https://github.com/istio/istio.github.io/pull/1497